### ISSUE TITLE
A prompt with multiple images?

### ISSUE THREAD

**Issue by LinB203** [JourneyDB/JourneyDB#2]
For example, the train split data has 4,453,193 images with 1,643,375 prompts.
It seems that a prompt will generate 3 images?

**Comment by LinB203** [comment]
Btw. I have filled the form just now. I wonder that when will receive the url to download dataset.

**Comment by KeqiangSun** [comment]
> For example, the train split data has 4,453,193 images with 1,643,375 prompts.
> It seems that a prompt will generate 3 images?

Yes, for each prompt, there might be more than one corresponding image.

**Comment by KeqiangSun** [comment]
> Btw. I have filled the form just now. I wonder that when will receive the url to download dataset.

Thanks for your interest. We will send the URL this week:)

**Comment by KeqiangSun** [comment]
Thanks again for your interest in JourneyDB and your patience to wait for its release. We have sent the email to users who have submitted the form. Please check it out. And for new users, you will get the download link as soon as you submit the form.

**Comment by LinB203** [comment]
> Thanks again for your interest in JourneyDB and your patience to wait for its release. We have sent the email to users who have submitted the form. Please check it out. And for new users, you will get the download link as soon as you submit the form.

Thanks, but I wonder that how to download by linux command, because my PC has no enough memory and I want to download to my server.

**Comment by KeqiangSun** [comment]
In your case, the mega-cmd tool might help: https://mega.io/zh-hant/cmd. Additionally, we are currently in the process of storing our data on multiple platforms to provide you with more options and flexibility.

**Comment by LinB203** [comment]
> In your case, the mega-cmd tool might help: https://mega.io/zh-hant/cmd. Additionally, we are currently in the process of storing our data on multiple platforms to provide you with more options and flexibility.

That's cool!

**Comment by LinB203** [comment]
> In your case, the mega-cmd tool might help: https://mega.io/zh-hant/cmd. Additionally, we are currently in the process of storing our data on multiple platforms to provide you with more options and flexibility.

And could you provide a small size version? such as 256 pixel. 1024 pixel is too expensive for me.

**Comment by KeqiangSun** [comment]
It is a good idea. We don't have a plan currently, but we will consider doing so.
